### ISSUE TITLE
MacOS - Several FAILS when Connect

### ISSUE THREAD

**Issue by GMtheONE** [spacemeowx2/switch-lan-play#100]
MacOS Catalina
SX OS 2.9.5
OFW 10.0.1

I install BREW without problem.
But i got some "fails" on the terminal when connecting on different servers. When ill try in Windows at the same network doesn't happen, any clue what to do?

This is the report. Thanks in advance
--------------------------------------------------------------------

Input the relay server address [ domain/ip:port ]:switch.lan-play.com:11451
Opening interfaces
[DEBUG]: open en0 ok
[DEBUG]: open p2p0 fail: datalink(12)
[DEBUG]: open awdl0 ok
[DEBUG]: open llw0 ok
[DEBUG]: open utun0 fail: datalink(0)
[DEBUG]: open utun1 fail: datalink(0)
[DEBUG]: open utun2 fail: datalink(0)
[DEBUG]: open utun3 fail: datalink(0)
[DEBUG]: open utun4 fail: datalink(0)
[DEBUG]: open utun5 fail: datalink(0)
[DEBUG]: open lo0 fail: datalink(0)
[DEBUG]: open bridge0 ok
[DEBUG]: open en1 ok
[DEBUG]: open en2 ok
[DEBUG]: open gif0 fail: datalink(0)
[DEBUG]: open stf0 fail: datalink(0)
[DEBUG]: open ap1 ok
pcap loop start
[DEBUG]: packet init buffer 0x105e0d5e0
Server IP: 149.91.81.201
[DEBUG]: uv_pcap_interf_sendpacket failed -1

**Comment by spacemeowx2** [comment]
most of the fails are ignorable. just one of the interface is needed to be opened

**Comment by GMtheONE** [comment]
So its ok? Perfect thank you!

**Comment by Roadhog360** [comment]
This is false, my friend cannot see my room even though he gets no errors and we're on the same server.

**Comment by drizuid** [comment]
> This is false, my friend cannot see my room even though he gets no errors and we're on the same server.

Please open your own issue with your own details, we have no clue what your issue is (or if it's your issue at all, it could be your friend)  Better yet, come on the switch lan play discord and get live help.
